EPRN.Portal/Profiles/PRNProfile.cs

```diff
@@ -13,6 +13,9 @@ public PRNProfile()
             CreateMap<StatusAndProducerDto, RequestCancelViewModel>()
                 .ForMember(d => d.Regulator, o => o.Ignore()); // we don't know where the regulator is coming from yet
             CreateMap<PrnDto, PrnRowViewModel>();
+            CreateMap<ViewDraftPrnViewModel, PrnDto>()
+                .ForMember(s => s.Status, opt => opt.Ignore())
```

Comment:
Is this really correct? We're ignoring status'? Despite having the value available to us in the database?

Reply:
In this particular view model I don't need the status. Separately, the records have already been filtered on status=draft.